.gitignore

```diff
@@ -1,3 +1,5 @@
 build
 composer.lock
 vendor
+psalm.xml
```

Comment:
I'm curious why the `psalm.xml`, the `Psalm` setting file is under `.gitignore` file.

Reply:
Fixed in https://github.com/ipinfo/php/pull/21/commits/d276dcae072d326c6d23ef78ed40310378badf8e